lib/iris/cube.py

```diff
@@ -1587,28 +1577,7 @@ def dtype(self):
         :class:`~iris.cube.Cube`.
 
         """
-        return self._data_manager.dtype
-
-    @property
-    def fill_value(self):
-        """
-        A fill value for the data of the :class:`~iris.cube.Cube`.
-
-        This is a value suitable for filling masked points in this cube's data.
-        It may be ``None``, meaning that no suitable fill value is known.
-
-        .. Note::
-
-            Ideally, this value will not occur anywhere in the cube data.
-            Thus, many operations that change cube data will also set
-            ``fill_value`` to ``None``.
-
-        """
-        return self._data_manager.fill_value
-
-    @fill_value.setter
-    def fill_value(self, fill_value):
-        self._data_manager.fill_value = fill_value
```

Comment:
@lbdreyer Yup, nice.

There's no point in even having a `fill_value` getter convenience, as it's not possible to get a lazy `fill_value` or indeed track whether the lazy array is an ndarray or masked without realising it (please tell me otherwise).